sbt/src/sbt-test/project/scripted-skip-incompatible/test

```diff
@@ -0,0 +1,17 @@
+# Setup failing test that should be skipped when tests are discovered
+$ copy-file changes/Fail.scala src/sbt-test/group/skipped/changes/Fail.scala
+$ copy-file changes/build.properties src/sbt-test/group/skipped/project/build.properties
+$ copy-file changes/test src/sbt-test/group/skipped/test
+
+# Should fail when run explicitly
+-> scripted group/skipped
```

Comment:
A scripted test running scripted, how meta. I love it 😄

Reply:
Copied from `sbt/src/sbt-test/project/scripted-plugin`. I had to wrap my head around it a few times to understand what was going on, but I think it makes perfect sense in this case to test with and without the `build.properties` file.